types/react-redux/index.d.ts

```diff
@@ -56,6 +56,8 @@ export type RootStateOrAny = AnyIfEmpty<DefaultRootState>;
 // Omit taken from https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-8.html
 export type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>;
 
+export type DistributiveOmit<T, K extends keyof T> = T extends unknown ? Omit<T, K> : never;
```

Comment:
I think this is technically *non*distributive, although I need to read the original bug report more closely to make sure.

Reply:
Well...I'm not really sure anymore. I guess if it fixes the bug, then it's OK.